utils/storage/store.js

```diff
@@ -1,5 +1,5 @@
 import defaults from "@ndhoule/defaults";
-import store from "@segment/store";
+import store from "storejs";
```

Comment:
`storejs` module doesn’t have `enabled` method to check local storage is accessible or not.
So instead of `store.enabled` add a checking as we have in the cookie to determine whether it is accessible.